sdk/src/local_device.h

```diff
@@ -36,6 +36,13 @@ class LocalDevice : public DeviceInterface {
     virtual aditof::Status applyCalibrationToFrame(uint16_t *frame,
                                                    const std::string &mode);
 
+    aditof::Status getDeviceFileDescriptor(int &fileDescriptor);
```

Comment:
these methods do not fall in the same category (`public: // implements DeviceInterface`) as the rest of the methods. You could add another public modifier for these calls to make things more clear.

Reply:
I have added another `public` modifier for these methods. Thanks!